Dockerfile

```diff
@@ -5,7 +5,12 @@
 # Use Node image, maintained by Docker:
 # hub.docker.com/r/_/node/
 FROM node:lts-alpine3.10
-WORKDIR /app
+
+# set working directory
+WORKDIR /forms-flow-forms/app
```

Comment:
this have to change as there is no forms-flow-forms folder in fork

Reply:
This folder is created inside Docker container image  right ? Should work with forms-flow-forms So could use Same. But changing to /app should also work